libbuteosyncfw/profile/SyncProfile.cpp

```diff
@@ -259,15 +259,11 @@ QDateTime SyncProfile::lastSuccessfulSyncTime () const
     if (d_ptr->iLog)
     {
         QList<const SyncResults*> allResults = d_ptr->iLog->allResults();
-        if (allResults.size () > 0)
-        {
-            lastSuccessSyncTime = allResults.at (0)->syncTime ();
```

Comment:
If the log has only one line ant this line is a fail sync. It will consider it as a successful sync.